Formula/libxml2.rb

```diff
@@ -41,7 +41,7 @@ class Libxml2 < Formula
 
   keg_only :provided_by_osx
 
-  depends_on :python => :optional
+  depends_on :python if MacOS.version <= :snow_leopard
```

Comment:
Hard request to not stick `python` with `keg_only` for as long as macOS refuse to link it to a recent crypto library 😭.